src/simplebar.js

```diff
@@ -188,8 +189,8 @@ export default class SimpleBar {
 
         this.el.insertBefore(this.trackX, this.el.firstChild);
         this.el.insertBefore(this.trackY, this.el.firstChild);
-
-		this.__isDomInitialized = true;
+        
+        this.__isDomInitialized = true;
```

Comment:
So I'm not sure why you had to do this but I think it's not needed in this PR at all? If so, can you remove it so I can merge the fix? Thanks